daisymfcgui/org.daisy.pipeline.lite/build.properties

```diff
@@ -0,0 +1,3 @@
+source.. = src/
+bin.includes = META-INF/,\
+               .
```

Comment:
Do we need the build.properties files? What are they for?

Reply:
Fixed when Nico fixed up commit https://github.com/daisy/pipeline1/commit/fba71f39fd7f4487978529b8c5bbd76730aa62f2 to https://github.com/daisy/pipeline1/commit/736915089efcc8f4e8a9a7c0c7885fe2746bda02.